README.md

```diff
@@ -60,14 +60,26 @@ listening for API requests and provisioning volumes accordingly.
 
 ### Architecture and Operation
 
-The PMEM-CSI driver can operate in two different DeviceModes: LVM and Direct
+The PMEM-CSI driver can operate in two different device modes LVM and Direct. This table contains an overview and comparison of those modes. There is more detailed modes explanation down in following paragraphs.
```

Comment:
Here "two different device modes LVM and Direct" needs to be "two different device modes: LVM and Direct".

Do we really want to capitalize "Direct"? LVM is upper case because it is an abbreviation, but "direct" is a normal English word.

Reply:
In plain text I would use "direct device mode" and "LVM device mode".